automation/test_containerdisk_annotations.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2022 Red Hat, Inc.
+#
+set -ex
+shopt -s extglob
+
+# Any template can be used here as they all have the same version label, fedora is an arbitrary one
+ver_label=$( \
+  yq -o props '.metadata.labels | with_entries(select(.key == "template.kubevirt.io/version"))' \
+  dist/templates/fedora-server-large.yaml \
+)
+
+templates=(dist/templates/+(centos7|centos-stream|fedora|rhel8|rhel9)-!(saphana)-*.yaml)
```

Comment:
what about others templates (ubutu, opensuse)?

Reply:
There are no containerdisks for those. (currently)